src/main/java/org/reaktivity/nukleus/tcp/internal/stream/ServerStreamFactory.java

```diff
@@ -75,8 +75,10 @@
     private final Function<RouteFW, LongConsumer> supplyWriteBytesAccumulator;
     private final Function<RouteFW, LongConsumer> supplyReadBytesAccumulator;
 
+    private final TcpConfiguration config;
```

Comment:
Not used, right?

Reply:
It is used to calculate windowThreshold below. But I will move that to the constructor and remove the config field.